docs/python/reference/configuration.md

```diff
@@ -24,29 +24,34 @@ class DBOSConfig(TypedDict):
     name: str
     database_url: Optional[str]
     sys_db_name: Optional[str]
-    app_db_pool_size: Optional[int]
     sys_db_pool_size: Optional[int]
     db_engine_kwargs: Optional[Dict[str, Any]]
     log_level: Optional[str]
     otlp_traces_endpoints: Optional[List[str]]
-    run_admin_server: Optional[bool]
+    otlp_logs_endpoints: Optional[List[str]]
     admin_port: Optional[int]
+    run_admin_server: Optional[bool]
 ```
 
 - **name**: Your application's name.
-- **database_url**: A connection string to a Postgres database. The supported format is
-```
-postgresql://[username]:[password]@[hostname]:[port]/[database name]
```

Comment:
Why did we remove this?

Reply:
Because we now support arbitrary connection strings, not the subset above, which had to be detailed. If you think this will help users, I can add it back